<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gamesetting</class>
 <widget class="QMainWindow" name="gamesetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_player2">
        <property name="title">
         <string>player2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="2" column="0">
            <widget class="QLabel" name="label_color_player2">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="comboBox_player2">
             <item>
              <property name="text">
               <string>blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>yellow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>green</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pink</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>purple</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>black</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>white</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_name_player2">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_player2"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_warning2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="btn_letsgo">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Lets go</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox_player1">
        <property name="title">
         <string>player1</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_name_player1">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_player1"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_color_player1">
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_player1">
             <item>
              <property name="text">
               <string>red</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>blue</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>yellow</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>green</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>pink</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>purple</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>black</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>white</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_warning1">
        <property name="styleSheet">
         <string notr="true">color: red;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
